controllers/messages.js

```diff
@@ -39,6 +39,8 @@ const getTaskAndDocForMessage = function (messageId, docs) {
       return [task, doc._id];
     }
   }
+
+  return [null,null];
```

Comment:
This is really ugly... why are we returning an array when the two items aren't the same? How about returning `{ task: task, _id: doc._id }`, and `{ task: null, _id: null }`? You can still destructure it on the other side with `const { task, docId }` but each component is labelled and obvious.